tests/test/scheduler/test_executor.cpp

```diff
@@ -330,6 +330,34 @@ TEST_CASE_METHOD(TestExecutorFixture,
     REQUIRE(restoreCount == 0);
 }
 
+TEST_CASE_METHOD(TestExecutorFixture,
+                 "Test executing function repeatedly and flushing",
+                 "[executor]")
+{
+    std::shared_ptr<BatchExecuteRequest> req =
+      faabric::util::batchExecFactory("dummy", "simple", 1);
+    uint32_t msgId = req->messages().at(0).id();
```

Comment:
Executing the same message over and over isn't really the intended use-case, I would put the creation of this request inside the loop so it's a different message every time.